### ISSUE TITLE
Use "International" weather data outside of NSRDB range in PVWatts call

### ISSUE THREAD

**Issue by adfarth** [NREL/REopt.jl#70]

**Comment by Bill-Becker** [comment]
@adfarth  can you confirm that this is done, and close it if so?

**Comment by brtietz** [comment]
FYI - since this issue was filed PVWatts added Europe and Africa data. I'd be surprised if there were a significant number of locations in NSRDB that aren't in PVWatts today.

**Comment by adfarth** [comment]
Yes confirmed this can be closed per this more recent PR: https://github.com/NREL/REopt.jl/pull/373 

"Change the way we determine which dataset to utilize in the PVWatts API call. Previously, we utilized defined lat-long bounds to determine if "nsrdb" or "intl" data should be used in PVWatts call. Now, we call the Solar Dataset Query API (v2) (https://developer.nrel.gov/docs/solar/data-query/v2/) to determine the dataset to use, and include "tmy3" as an option, as this is currently the best-available data for many locations in Alaska." 

And @brtietz --that's awesome that the PVWatts data was updated and I'm very glad we preemptively moved away from lat-long bounds to define the NSRDB region given the evolution of the dataset!
